### DISCUSSION TITLE
Next completion does not work in CLion 2022.3 EAP Copilot 1.1.35.2063

### DISCUSSION THREAD

**Discussion by andrewgazelka** [community/community#38109] [Copilot Conversations]
When using the "Show Next Completions" keybind **⌥ ]** (and as well as selecting it in manually), Copilot does not go to the next completion—the first completion is always selected.

There are no overriding keymaps
<img width="1548" alt="image" src="https://user-images.githubusercontent.com/7644264/199607805-e2d57644-11e6-4578-ac5b-59c8dd2f0408.png">

Me selecting it manually
<img width="358" alt="image" src="https://user-images.githubusercontent.com/7644264/199608128-f93f4080-c7c6-41f5-b8c4-cfcd2aaa45f0.png">

The issue also occurs in nightly Copilot as of Wed, Nov 2, 2022

**Comment by ossanna16** [comment]
Hi there @andrewgazelka and welcome to our community! Thank you for reporting this!

**Comment by iamcjwilliams** [comment]
@andrewgazelka Our copilot team is investigating this

**Reply by andrewgazelka** [reply]
> @andrewgazelka Our copilot team is investigating this

It seems to work now. It might be related to `Grazie`, as disabling all Grazie features seemed to fix it. I don't know if this is a Grazie or Copilot issue.
